src/libraries/Native/AnyOS/System.Globalization.Native/pal_calendarData.c

```diff
@@ -5,12 +5,19 @@
 #include <assert.h>
 #include <stdlib.h>
 #include <string.h>
+
+#ifdef TARGET_UNIX
 #include <strings.h>
+#endif
 
 #include "pal_locale_internal.h"
 #include "pal_errors_internal.h"
 #include "pal_calendarData.h"
 
+#ifndef TARGET_UNIX
+#define strcasecmp _stricmp
```

Comment:
You could perhaps consider moving this define into the first ifdef like this:

```
#ifdef TARGET_UNIX
#include <strings.h>
#else
#define strcasecmp _stricmp
#endif
```
(But I'm not sure if perhaps the intent was to only scope it to this file alone.)

Reply:
Yeah since this is the only file that currently uses it I wanted to only scope it to this file. If we need to share it, We could move it to pal_compiler.h or some shared header like pal_icushim_internal.h

Reply:
Do the other included headers use anything from `strings.h` ?
If not, I wonder if maybe it could be moved below the other includes to really scope it to only this file.
(FWIW, strings.h isn't standard, so reducing usage of it would probably be good)

---
In reply to: [394643751](https://github.com/dotnet/runtime/pull/33704#discussion_r394643751) [](ancestors = 394643751)

Reply:
Ah right. Good idea. I don't think they do.